evaluation/eval.py

```diff
@@ -63,14 +98,18 @@ def main():
     tokenizer = AutoTokenizer.from_pretrained(eval_args.tokenizer_name or eval_args.model_name_or_path)
     tokenizer.pad_token = tokenizer.eos_token
     tokenizer.padding_side = "left"
-
-    model = AutoModelForCausalLM.from_pretrained(
+    if "T0" in eval_args.model_name_or_path:  # in ["bigscience/T0_3B", "bigscience/T0"]:
+        MODEL_TYPE = AutoModelForSeq2SeqLM
```

Comment:
I'd recommend renaming the variable `auto_model` and making it lowercased local variable. 
```suggestion
        auto_model = AutoModelForSeq2SeqLM
```